tests/io/v3/vm/scanner_groups/test_scanner_group_api.py

```diff
@@ -0,0 +1,182 @@
+'''
+Testing the Scanner groups endpoints
+'''
+import re
+
+import responses
+from responses import matchers
+
+SCANNER_GROUPS_BASE_URL = r'https://cloud.tenable.com/api/v3/scanner-groups'
+GROUP_ID = 'b5db63f1-551d-4789-aefa-9629c93ddc45'
+SCANNER_ID = 'b5db63f1-551d-4789-aefa-9629c93ddc45'
+SCANNER_G_DETAILS = {
```

Comment:
Could you please write the constants name as complete. It will be easier for others.